quickstart/examples/inference-scheduling/ms-inference-scheduling/values_tpu.yaml

```diff
@@ -0,0 +1,123 @@
+# This values.yaml file creates the resources for Qwen/Qwen3-0.6B
+multinode: false
+
+modelArtifacts:
+  uri: "hf://meta-llama/Llama-3.1-70B"
```

Comment:
Is there a reason to use a different model other than the Qwen/Qwen3-0.6B model for TPU?